### ISSUE TITLE
create a disabled category

### ISSUE THREAD

**Issue by JLZIMMERMANN** [plepe/OpenStreetBrowser#155]
It would be welcome to create a disabled category that would stimulate contributions for mappers that are wishing to improve data for people that are in wheelchair, blind, deaf, cognitive disability.

**Comment by plepe** [comment]
Sure, sounds great. Do you think these would belong in the normal categories or is this more for Quality Assurance (a few categories for this will be the next update)?

Any ideas what queries this category should use? Could you try to build a custom category as basis? https://blog.openstreetbrowser.org/node/86

**Comment by JLZIMMERMANN** [comment]
Maybe a specific category Disabilities would be welcome to let query each : deaf, wheelchair, blind, mental disability...

**Comment by plepe** [comment]
I'm still not sure how this should develop. I made a first attempt at a wheelchair-accessibility category for shop/craft/amenity: https://openstreetbrowser.org/#categories=custom/03484db201a347eb00166948f1333123

What do you think? (Click the (i) for the map key)

**Comment by JLZIMMERMANN** [comment]
Great starting point !
![image](https://user-images.githubusercontent.com/4929752/186511140-85be4ba4-6dd7-47c3-85de-5fea2ac9681a.png)

Could you do the same for entrances, kerb, crossing and sidewalks (way) ? Using the same method green (yes),yellow (limited,bad,half) ,red (no)

**Comment by plepe** [comment]
An example for Kerbs: https://openstreetbrowser.org/#categories=custom/00443660c0570c2851cbc9f119019ad5
How do you like it?

Of course, when they are finished, I would integrate them to the main categories ...

**Comment by JLZIMMERMANN** [comment]
I love ! It's very clever : even there are no wheelchair=yes in case of flush or lowered you creaded some near green colour.
Impressive good idea ;-)
![image](https://user-images.githubusercontent.com/4929752/186520927-388ae98a-c580-4e8d-baca-807ed9e36bf7.png)

**Comment by plepe** [comment]
I love it, when they are on ways:
![image](https://user-images.githubusercontent.com/249012/186522334-4f605886-9797-4289-b16a-4c07477d0ee4.png)

Btw, the colors are similar to those on https://wiki.openstreetmap.org/wiki/Key:kerb

**Comment by JLZIMMERMANN** [comment]
Maybe it would be interesting to test a line around the dots when the kerb has wheelchair=*

**Comment by plepe** [comment]
Kerbs with wheelchair lines: https://openstreetbrowser.org/#categories=custom/174a6c1455638cb6ebba922c6108c11e

If not set, I'm using white - I could also use the implied values (e.g. lowered => wheelchair=yes)

Next, it would be interesting to add tactile_paving, maybe as a fill pattern. Unfortunately, I haven't added fill patterns to OpenStreetBrowser yet, so this will take some time ...

**Comment by JLZIMMERMANN** [comment]
Well done, the outside line will let us understand complexity of answers in city for disabled people.
I tried in 2013 to do something for this when I worked in a municipality it was Lizmobility. This tool does not exist anymore because the municipality didn't pay anymore and I left my job there. [A link](https://fr.slideshare.net/datatuesday/10-jeanlouis-zimmermann-open-streetmap-france-lizmobility)

**Comment by plepe** [comment]
It would be interesting to choose the disability; right now the OpenStreetBrowser system is not ready for that yet (a category can't have a config section). I was planning to do so for some time, so I guess I'll do this soon (which might still take a few months).

Anyway, I'd like to publish the Kerb category soon (although without the wheelchair display, and add this when the configuration is finished). Do you think I should make this a sub-category of Transportation/Walking or Infrastructure/Roads?

**Comment by JLZIMMERMANN** [comment]
Yes add this experiment into walking is good. I haven't seen sidewalks.

**Comment by plepe** [comment]
You are right, if I publish Kerbs, I should publish Sidewalks as well.

Sidewalks are quite complicated. I made an attempt here, what do you think? https://openstreetbrowser.org/#categories=custom/6a421d07e65563f7cb3d761c8037d3b2
![image](https://user-images.githubusercontent.com/249012/187369505-47d899ae-741a-48fb-ae5e-bffc72329b8a.png)

**Comment by JLZIMMERMANN** [comment]
Very good improvement. 
It may be missing ramps,
I'm wandering to show also entrances (square dot like OSM rendering) that are connected and disabled parking space as small dots.
Maybe highlight all lines like you did (green, yellow, red).
